Gabriellji/Express-SQL-REST-API
# Routes

### Implemented following features:
+ Unit tests (Supertest, jest)
+ GET - Retrieve all of the data from a table
+ GET - Retrieve specific fields (i.e. id, names, dates, etc.)
+ GET - Retrieve a data set with the following filters :
-A filter for data that contains... (e.g. name containing the string 'wcs')
-A filter for data that starts with... (e.g. name beginning with 'campus')
-A filter for data that is greater than... (e.g. date greater than 18/10/2010)
+ GET - Ordered data recovery (i.e. ascending, descending) - The order should be passed as a route parameter
+ POST - Insertion of a new entity
+ PUT - Modification of an entity
+ PUT - Toggle a Boolean value
+ DELETE - Delete an entity
+ DELETE - Delete all entities where boolean value is false
